### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by trodge**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
 
+FEATURES:
+* **New Resource:** `google_cloudbuildv2_connection` (ga) ([#15098](https://github.com/hashicorp/terraform-provider-google/pull/15098))
+* **New Resource:** `google_cloudbuildv2_repository` (ga) ([#15098](https://github.com/hashicorp/terraform-provider-google/pull/15098))
+* **New Resource:** `google_gkeonprem_bare_metal_admin_cluster` (ga) ([#15099](https://github.com/hashicorp/terraform-provider-google/pull/15099))
+* **New Resource:** `google_network_security_address_group` (ga) ([#15111](https://github.com/hashicorp/terraform-provider-google/pull/15111))
+* **New Resource:** `google_network_security_gateway_security_policy_rule` (ga) ([#15112](https://github.com/hashicorp/terraform-provider-google/pull/15112))
+* **New Resource:** `google_network_security_gateway_security_policy` (ga) ([#15112](https://github.com/hashicorp/terraform-provider-google/pull/15112))
+* **New Resource:** `google_network_security_url_lists` (ga) ([#15112](https://github.com/hashicorp/terraform-provider-google/pull/15112))
+* **New Resource:** `google_network_services_gateway` (ga) ([#15112](https://github.com/hashicorp/terraform-provider-google/pull/15112))
```
Comment (shuyama1) [inline CHANGELOG.md:11]:
```suggestion
* **New Resource:** `google_cloudbuildv2_connection` ([#15098](https://github.com/hashicorp/terraform-provider-google/pull/15098))
* **New Resource:** `google_cloudbuildv2_repository` ([#15098](https://github.com/hashicorp/terraform-provider-google/pull/15098))
* **New Resource:** `google_gkeonprem_bare_metal_admin_cluster` ([#15099](https://github.com/hashicorp/terraform-provider-google/pull/15099))
* **New Resource:** `google_network_security_address_group` ([#15111](https://github.com/hashicorp/terraform-provider-google/pull/15111))
* **New Resource:** `google_network_security_gateway_security_policy_rule` ([#15112](https://github.com/hashicorp/terraform-provider-google/pull/15112))
* **New Resource:** `google_network_security_gateway_security_policy` ([#15112](https://github.com/hashicorp/terraform-provider-google/pull/15112))
* **New Resource:** `google_network_security_url_lists` ([#15112](https://github.com/hashicorp/terraform-provider-google/pull/15112))
* **New Resource:** `google_network_services_gateway` ([#15112](https://github.com/hashicorp/terraform-provider-google/pull/15112))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+
+IMPROVEMENTS:
+* bigquery: added `storage_billing_model` argument to `google_bigquery_dataset` ([#15115](https://github.com/hashicorp/terraform-provider-google/pull/15115))
+* bigquery: added `external_data_configuration.metadata_cache_mode` and `external_data_configuration.object_metadata` to `google_bigquery_table` ([#15096](https://github.com/hashicorp/terraform-provider-google/pull/15096))
+* bigquery: made `external_data_configuration.source_fomat` optional in `google_bigquery_table` ([#15096](https://github.com/hashicorp/terraform-provider-google/pull/15096))
+* certificatemanager: added `issuance_config` field to `google_certificate_manager_certificate` resource ([#15101](https://github.com/hashicorp/terraform-provider-google/pull/15101))
+* cloudbuild: added `repository_event_config` field to `google_cloudbuild_trigger` resource (ga) ([#15098](https://github.com/hashicorp/terraform-provider-google/pull/15098))
+* cloudbuildv2: added `gitlab_config` field to `google_cloudbuildv2_connection` resource (beta) ([#15053](https://github.com/hashicorp/terraform-provider-google/pull/15053))
```
Comment (shuyama1) [inline CHANGELOG.md:19]:
```suggestion
* cloudbuild: added `repository_event_config` field to `google_cloudbuild_trigger` resource ([#15098](https://github.com/hashicorp/terraform-provider-google/pull/15098))
```
We can remove beta changes from GA provider

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,40 @@
...
+* compute: added field `http_keep_alive_timeout_sec` to resource `google_compute_target_http_proxy` ([#15109](https://github.com/hashicorp/terraform-provider-google/pull/15109))
+* compute: added field `http_keep_alive_timeout_sec` to resource `google_compute_target_https_proxy` ([#15109](https://github.com/hashicorp/terraform-provider-google/pull/15109))
+* compute: added support for updating labels in `google_compute_external_vpn_gateway` ([#15134](https://github.com/hashicorp/terraform-provider-google/pull/15134))
+* container: made `monitoring_config.enable_components` optional on `google_container_cluster`(ga) ([#15131](https://github.com/hashicorp/terraform-provider-google/pull/15131))
```
Comment (shuyama1) [inline CHANGELOG.md:23]:
```suggestion
* container: made `monitoring_config.enable_components` optional on `google_container_cluster` ([#15131](https://github.com/hashicorp/terraform-provider-google/pull/15131))
```

**Review by shuyama1 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+* sql: added `settings.data_cache_config` to `sql_database_instance` resource. ([#15127](https://github.com/hashicorp/terraform-provider-google/pull/15127))
+* sql: added `settings.edition` field to `sql_database_instance` resource. ([#15127](https://github.com/hashicorp/terraform-provider-google/pull/15127))
+* vertexai: supported `shard_size` in `google_vertex_ai_index` ([#15133](https://github.com/hashicorp/terraform-provider-google/pull/15133))
+
+BUG FIXES:
+* compute: made `google_compute_router_peer.peer_ip_address` optional ([#15095](https://github.com/hashicorp/terraform-provider-google/pull/15095))
+* redis: fixed issue with `google_redis_instance` populating output-only field `maintenance_schedule`. ([#15063](https://github.com/hashicorp/terraform-provider-google/pull/15063))
+* monitoring: fixed an issue causing `google_monitoring_monitored_project` to appear to be deleted ([#15140](https://github.com/hashicorp/terraform-provider-google/pull/15140))
```
Comment (melinath) [inline CHANGELOG.md:33]: resolved in patch 
```suggestion
```

**Review by melinath — APPROVED** [review]

**Pull request merged**